### PR TITLE
limit POST /times

### PR DESCRIPTION
**Pull request by tschuy**

### CODE REVIEW

File: tests/times.js [hunk 1]
```diff
@@ -199,14 +199,48 @@ module.exports = function(expect, request, baseUrl) {
...
+          error: 'Authorization failure',
+          status: 401,
+          text: 'thai is not authorized to create times for project gwm.',
+        };
+
+        expect(res.statusCode).to.equal(401);
+        expect(body).to.deep.equal(expectedResult);
+
+        request.get(baseUrl + 'times', function(getErr, getRes, getBody) {
+          expect(getErr).to.deep.equal(null);
```
Comment (athai) [inline tests/times.js]: Why ``.to.deep.equal(null)`` and not ``.to.be.a('null')``?

**Comment by LyonesGamer** [comment]
+1 Test looks good.

**Comment by athai** [comment]
+1

**Comment by athai** [comment]
No errors that I can see and everything passes! +1

**Comment by LyonesGamer** [comment]
Looks good. +1

**Pull request merged**